### PR TITLE
fix(bridge-ui): update abis

### PR DESCRIPTION
**Pull request by jscriptcoder**
- Updated ABIs
- Using wagmi/cli to copy ABIs from protocol/abi (`pnpm export:abi`)
  - Generates `index.ts` exporting ABIs
- Increase coverage on recommendedProcessingFee

TODO: run exporting ABIs (`pnpm export:abi`) in the script before copying the json files?

### CODE REVIEW

**Review by dantaik — APPROVED** [review]

**Review by dantaik — DISMISSED** [review]

File: packages/bridge-ui/scripts/copy-abi.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```
Comment (dantaik) [inline packages/bridge-ui/scripts/copy-abi.js:6]: should we deleted all old files before copying?

Comment (jscriptcoder) [reply packages/bridge-ui/scripts/copy-abi.js:6]: They'll be overwritten, but yes, that's a good idea in case of change of name

**Comment by dantaik** [comment]
> * Updated ABIs
> * Script to copy ABIs from protocol/abi (`pnpm export:abi`)
>   
>   * Generates `index.ts` exporting ABIs
> * Increase coverage on recommendedProcessingFee
> 
> TODO: run exporting ABIs (`pnpm export:abi`) in the script before copying the json files?

Maybe we should add a github hook to automate the abi export?

File: packages/bridge-ui/scripts/copy-abi.js
[the remark this answers is not in the document]
Comment (d1onys1us) [reply packages/bridge-ui/scripts/copy-abi.js:1]: if it's a low hanging fruit like short task then i'd say we can try it. but if it takes too much time i agree let's just go with this custom script.

**Review by dionysuzx — DISMISSED** [review]

**Review by dantaik — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Pull request merged**